common/src/main/java/org/vivecraft/client_vr/gameplay/trackers/ClimbTracker.java

```diff
@@ -109,7 +110,7 @@ public boolean isActive(LocalPlayer p) {
     }
 
     public boolean isClimbeyClimb() {
-        return this.isActive(this.mc.player) && this.isClimbeyClimbEquipped();
+        return this.isActive(this.mc.player) && this.isClimbeyClimbEquipped() && (!OriginsHelper.isLoaded() || OriginsHelper.hasClimbingPower(this.mc.player));
```

Comment:
this should be 
```
return this.isActive(this.mc.player) && (this.isClimbeyClimbEquipped() || (OriginsHelper.isLoaded() && OriginsHelper.hasClimbingPower(this.mc.player)))
```

Reply:
Oh yea ur right, whoops

Reply:
I'm actually gonna move it to isClimbeyClimbEquipped cause there's other checks that need it.